src/modules/display-comments.js

```diff
@@ -0,0 +1,38 @@
+async function displayCommentsList(id) {
```

Comment:
As per the project requirements, you should be using ES6 features, that include the use of arrow functions instead of the old function declaration way. Kindly update your code accordingly.